manifests/g/gerardog/gsudo/portable/2.4.2/gerardog.gsudo.portable.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: gerardog.gsudo.portable
+PackageVersion: 2.4.2
+InstallerLocale: en-US
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: net46-AnyCpu/gsudo
+  PortableCommandAlias: gsudo
```

Comment:
Should have been:

- RelativeFilePath: net46-AnyCpu/gsudo.exe
  PortableCommandAlias: gsudo.exe

Reply:
I initially tried to link shell script for wsl, but that did not go well as winget symlinks everything as .exe.